tests/src/main/java/ca/bc/gov/open/jagefilingapi/qa/frontendutils/JsonReader.java

```diff
@@ -0,0 +1,38 @@
+package ca.bc.gov.open.jagefilingapi.qa.frontendutils;
+
+import com.google.gson.JsonArray;
+import com.google.gson.JsonElement;
+import com.google.gson.JsonObject;
+import com.google.gson.JsonParser;
+
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.util.Map;
+
+public class JsonReader {
+
+    public static Object[][] searchJsonElement(JsonArray jsonArray, int totalDataRow, int totalColumnEntry) {
```

Comment:
question: what is this doing??

Reply:
For sending multiple test datasets in the future, treating datasets in the array as rows and each key/value pairs as columns like excel and iterating through the json array to parse. Hope that makes sense.

Reply:
yeah it does, just wondering if/how we can simplify this

Reply:
2D array was approach was to work with the @dataprovider annotaion. Refactored now. Thank you.